libraries/abstractions/transport/utest/transport_secure_sockets_utest.c

```diff
@@ -0,0 +1,759 @@
+/*
+ * FreeRTOS Secure Sockets V1.2.0
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+#include "unity.h"
+
+#include "mock_iot_secure_sockets.h"
+
+/* Transport interface include. */
+#include "transport_interface.h"
+#include "transport_secure_sockets.h"
+#include "iot_secure_sockets.h"
+
+/* The send and receive timeout to set for the socket. */
+#define SEND_RECV_TIMEOUT          0
+
+/* The host and port from which to establish the connection. */
+#define HOSTNAME                   "amazon.com"
+#define INVALID_HOSTNAME           "amazon.com9999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999"
+#define PORT                       80
+
+/* Configuration parameters for the TLS connection. */
+#define MFLN                       42
+#define ALPN_PROTOS                "x-amzn-mqtt-ca"
+
+#define MOCK_ROOT_CA               "mockRootCA"
+#define MOCK_SERVER_ADDRESS        100
+#define MOCT_TCP_SOCKET            100
+#define MOC_SECURE_SOCKET_ERROR    -1
+
+
+/* Parameters to pass to #SecureSocketsTransport_Send and #SecureSocketsTransport_Recv. */
+#define BYTES_TO_SEND                      4
+#define BYTES_TO_RECV                      4
+#define SECURE_SOCKETS_READ_WRITE_ERROR    -1
+
+/* The size of the buffer passed to #SecureSocketsTransport_Send and #SecureSocketsTransport_Recv. */
+#define BUFFER_LEN                         4
+
+/**
+ * @brief Transport timeout in milliseconds for transport send.
+ */
+#define TEST_TRANSPORT_SND_TIMEOUT_MS      ( 5000U )
```

Comment:
To be consistent with the above macros, can you update the above macros to be enclosed in parenthesis ( `(` `)`)?

Reply:
made it consistent in de398277dbb6eede0480df8a1a53b7a6515ca4b6